examples/guidelines-snap-tutorials.md

```diff
@@ -36,30 +23,95 @@ Each tutorial should:
+The tone of your tutorial should be friendly. Try to make the reader feel that they're building and learning something together with you.
 
-Also, be conscious that we want all tutorials to feel the same, cross-topics. This is why you should perform one or two tutorials first before starting writing one, to match the tone we are trying to establish.
+All tutorials should have the same tone, regardless of the topic. This is why you should complete one or two of the existing tutorials before writing your first one.
 
-In short, this isn’t a teacher/student paradigm, but rather friends sharing some time together. Thus, should use the term **we** as much as possible, like **we have just seen**, **we now understand that…**. However **you** can be use for demonstrating things in the user’s context, like: **edit your `file`**, **your `directory` should look like this:**, **your system** or **you will note**. Prefer the **we** as much as possible though.
+In short, this isn’t a teacher/student paradigm, but rather friends sharing some time together. Thus, the term **we** should be used as much as possible, like **we have just seen**, **we now understand that…**. However **you** can be use for demonstrating things in the user’s context, like: **edit your `file`**, **your `directory` should look like this:**, **your system** or **you will note**.
 
 And now, let's see the first required step!
 
+## Metadata
+Duration: 2:00
+
+Each tutorial source file starts with a set of metadata that will be consumed by tutorials.ubuntu.com for presentation purposes.
+
+For example, at the top of the source file of this tutorial:
+
+```
+---
+id: tutorial-guidelines
+summary: A summary tutorial description. Remember the content of it (as the title) is searchable. It should be around 26 words.
+categories: snapcraft
+tags: tutorial,guidelines,snap
+difficulty: 2
+status: Published
+author: Javier Lopez <javier.lopez@example.com>
+published: 2017-01-13
+
+---
+```
+
+ * **id** is used as tutorial url
+ * **categories** correspond to one or multiple areas of the developer site
+ * Remember that **tags** are searchable
+
+
+ * **difficulty** spans from 1 to 5. Beginners without previous knowledge of the given topic should be able to follow and understand tutorials from level 1 without any other prerequisite knowledge. As a guide:
+
+     1. Complete beginner with Ubuntu - just about knows how to open a terminal
+     * Ubuntu novice - can be trusted to enter commands but experience limited to simple file operations.
+     * Experienced user - doesn't need explanations about common topics (networking, sudo, services)
+     * Advanced user - has experience running Ubuntu for many years, but may be unfamiliar with some sysadmin/programming topics.
+     * Ubuntu sysadmin/developer- very familiar with system and can be expected to know details about
```

Comment:
very familar with *the* system*? Also, this is a ltitle vague. Perhaps very familiar with all aspects of the operating system ...